src/graph.c

```diff
@@ -311,7 +311,7 @@ static void check_reachability(void) {
 }
 
 void graph(void) {
-	subnet_cache_flush();
+	subnet_cache_flush_table(SUBNET_MAC);
```

Comment:
Even though it does the correct thing, this just looks like it will only flush the MAC address cache. It's better to have an explicit `subnet_cache_flush_all_tables()` function.

Reply:
I've updated it to use `SUBNET_TYPES` so it's clearer.